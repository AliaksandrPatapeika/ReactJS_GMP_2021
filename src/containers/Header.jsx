import '../less/containers/Header.less';

import React from 'react';

import HeaderMain from './HeaderMain';
import HeaderTop from './HeaderTop';

const Header = () => (
  <header className="headerContainer">
    <div className="headerBlurImageLayer">
      <HeaderTop />
      <HeaderMain />
    </div>
  </header>
);

export default Header;
