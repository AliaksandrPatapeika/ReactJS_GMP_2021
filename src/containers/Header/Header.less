.headerContainer {
	height: 417px;
	margin-bottom: 10px;
	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('../../img/bgImageNew.jpg');
	background-size: cover;

	.headerBlurImageLayer {
		height: 100%;
		backdrop-filter: blur(3px);
	}
}