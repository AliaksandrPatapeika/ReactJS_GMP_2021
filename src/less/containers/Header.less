.headerContainer {
	background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url('../../img/bgImageNew.jpg');
	background-size: cover;
	height: 417px;
	margin-bottom: 10px;

	.headerBlurImageLayer {
		backdrop-filter: blur(3px);
		height: 100%;
	}
}