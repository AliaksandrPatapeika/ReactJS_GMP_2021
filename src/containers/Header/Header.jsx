import './Header.less';

import React from 'react';

import HeaderMain from '../HeaderMain';
import HeaderTop from '../HeaderTop';

const Header = () => (
  <header className="headerContainer">
    <div className="headerBlurImageLayer">
      <HeaderTop />
      <HeaderMain />
    </div>
  </header>
);

export default Header;
