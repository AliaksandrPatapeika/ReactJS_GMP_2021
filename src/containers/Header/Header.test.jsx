import React from 'react';
import renderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {
  test('renders correctly', () => {
    const header = renderer
      .create(<Header />)
      .toJSON();

    expect(header).toMatchSnapshot();
  });
});
