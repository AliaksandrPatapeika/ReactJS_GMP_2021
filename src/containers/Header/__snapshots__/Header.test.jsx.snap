// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header
  className="headerContainer"
>
  <div
    className="headerBlurImageLayer"
  >
    <div
      className="headerTopContainer"
    >
      <div
        className="logoContainer"
      >
        <span
          className="netflix"
        >
          netflix
        </span>
        <span
          className="roulette"
        >
          roulette
        </span>
      </div>
      <button
        className="addMovieButton"
        onClick={[Function]}
        type="button"
      >
        + ADD MOVIE
        
      </button>
    </div>
    <div
      className="headerMainContainer"
    >
      <h1
        className="headerTitle"
      >
        FIND YOUR MOVIE
      </h1>
      <div
        className="searchContainer"
      >
        <input
          className="searchInput"
          placeholder="What do you want to watch?"
        />
        <button
          className="searchButton"
          onClick={[Function]}
          type="button"
        >
          SEARCH
          
        </button>
      </div>
    </div>
  </div>
</header>
`;
